lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -233,6 +233,8 @@ def create_or_update_user_script_phases(script_phases, native_target)
               phase.shell_path = script_phase[:shell_path] || '/bin/sh'
               phase.input_paths = script_phase[:input_files]
               phase.output_paths = script_phase[:output_files]
+              phase.input_file_list_paths = script_phase[:input_file_lists]
+              phase.output_file_list_paths = script_phase[:output_file_lists]
```

Comment:
should there be a class/model for the script phase? seems like it's getting big enough to warrant one

Reply:
probably...do we want it for this PR?

Reply:
Also what would it give us? There is no state to manage

Reply:
better type safety, generated docs, better IDE integration, etc.

don't think it's necessary for this PR though